sdk/search/azure-search-documents/azure/search/documents/_service/_models.py

```diff
@@ -0,0 +1,105 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from ._generated.models import Analyzer, Tokenizer
+
+class PatternAnalyzer(Analyzer):
+    """Flexibly separates text into terms via a regular expression pattern.
+    This analyzer is implemented using Apache Lucene.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :param odata_type: Required. Identifies the concrete type of the analyzer.Constant filled by
```

Comment:
Not sure to what "analyzer.Constant" refers. If it's a constant, how is it assigned by the server or anything? Maybe just a typo?

Also, if it's filled by the server, why is the user having to pass it as a required parameter?

Reply:
These are auto-generated code based on swagger (including the summary). Maybe it is better to keep them as much as possible?

Reply:
Not necessarily, but I would not block on this. I've actually changed quite a bit of the documentation for .NET. Sometimes the swagger just needs to be updated, but other changes have been idiomatic and more descriptive of our implementation. Our generator can merge customized code with generated code, so it makes it a little easier.

Reply:
Every "Analyzer" has this field. We need this information to know which type of analyzer it is.

Reply:
I think the misunderstanding here is that because of the missing space, @heaths got it as one token "analyzer.Constant", when it actually is: `Identifies the concrete type of the analyzer. Constant filled by server.` :)

I would document odata_type as a `ivar` at best, or maybe not at all (it's a constant, we don't even read the kwargs)